frontend/JavaScript/category.js

```diff
@@ -0,0 +1,43 @@
+window.onload = function(){
+fetch(`http://127.0.0.1:7002/categoryTree`,{
+    method : 'GET',
+    mode :'cors',
+    headers:{
+      'Access-Control-Allow-Origin':'*',
+      'Accept': 'application/json',
+      'Content-Type': 'application/json;charset=utf-8',
+      
+    }
+    
+  }).then(response => response.json()).then(data =>{
+    cat1 = data[0];
```

Comment:
Add null exception for the data being extracted from the response . If data is not received from api the UI will break here.

Reply:
Null exception has been handled for all the .js files.